tenantrate: update default rate_limit to -400

andy-kimball opened the pull request:
Previously, the default tenant rate limit was a relative value of -200, meaning that a single tenant is allowed to use up to ~20% of a KV node's CPU. This change updates that to -400, because experience has shown that it provides better tenant burst performance while still providing sufficiently predictable performance, given today's host cluster sizes and utilization.

Fixes: https://cockroachlabs.atlassian.net/browse/CC-25687

Release note: None

cockroach-teamcity commented:
[![CLA assistant check](https://cla.crdb.dev/pull/badge/not_signed)](https://cla.crdb.dev/cockroachdb/cockroach?pullRequest=110866) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.crdb.dev/cockroachdb/cockroach?pullRequest=110866) before we can accept your contribution.<br/><hr/>**Andy Kimball** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.crdb.dev/check/cockroachdb/cockroach?pullRequest=110866) it.</sub>

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/110866)
<!-- Reviewable:end -->

andy-kimball committed:
tenantrate: update default rate_limit to -400

Previously, the default tenant rate limit was a relative value of
-200, meaning that a single tenant is allowed to use up to ~20%
of a KV node's CPU. This change updates that to -400, because
experience has shown that it provides better tenant burst
performance while still providing sufficiently predictable
performance, given today's host cluster sizes and utilization.

Fixes: https://cockroachlabs.atlassian.net/browse/CC-25687

Release note: None

jeffswenson reviewed (approved):
LGTM

andy-kimball commented:
bors r+
